### ISSUE TITLE
xf:output mediatype="image/*" generates an <img> with no alt="…"

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2215]
This is an issue per WCAG2 1.1.1 (A), and is reported by TotalValidator as E860.

Solution: put the value of the label in the `alt`. We need to support this on page generation, and when the label changes after the page is loaded.
